src/Application.php

```diff
@@ -121,7 +121,7 @@ public function getConfig()
 
     public function setDir($dirname, $path)
     {
-        $this->dir[$dirname] = $path;
+        $this->dirs[$dirname] = $path;
```

Comment:
👍 
I made a test which reproduce this bug https://github.com/mercari/dietcube/pull/28
PR #24 will fix test failure of #28.